http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/BuildHelmChartValues.scala

```diff
@@ -245,7 +245,7 @@ private[leonardo] object BuildHelmChartValues {
                                                stagingBucket: GcsBucketName,
                                                customEnvironmentVariables: Map[String, String]
   ): List[String] = {
-    val rstudioIngressPath = s"/proxy/google/v1/apps/${cluster.cloudContext.asString}/${appName.value}/rstudio-service"
+    val rstudioIngressPath = s"/proxy/google/v1/apps/${cluster.cloudContext.asString}/${appName.value}/app"
```

Comment:
Yeah so I imagine we'll need to add a `buildSasAppChartOverrideValuesString` method, right?

It would be so nice if we had a common `buildAppChartOverrideValuesString` with a standard set of Leo values, and app charts built to that spec…

Reply:
yea for sure...definitely future optimization...
It'll be more clear how we can optimize once we attempt adding SAS.